host/xtest/symm_cipher_perf.c

```diff
@@ -662,14 +735,18 @@ int aes_perf_runner_cmd_parser(int argc, char *argv[])
 		}
 	}
 
+	algo = get_symm_algo(cipher, mode);
+	if (algo == -1)
+		return 1;
```

Comment:
This is not awaited. I think `assert(algo != -1)` is enough.
Otherwise, there should be an error trace message, something like `"Implementation is bugged, unexpected algo ID"`. I'm more in favor of the `assert()`.

Reply:
solved